Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "dcap_attestation"
+version = "0.1.0"
+edition = "2018"
```

Comment:
We can change `edition = "2021"` to the latest Rust edition. Can also change in the other Cargo.toml files.